CleanWeatherTests/Extensions/ErrorExtensionTests.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  ErrorExtensionTests.swift
+//  CleanWeatherTests
+//
+//  Created by Marek Skrzelowski on 28/11/2019.
+//  Copyright © 2019 vandermesis. All rights reserved.
+//
+
+import Quick
+import Nimble
+
+@testable import CleanWeather
+
+final class ErrorExtensionTests: QuickSpec {
+
+    override func spec() {
+
+        var error: Error!
+
+        describe("error user friendly message") {
```

Comment:
userfriendlymessage is using localizable strings, it'll fail if you get other translations, this whole describe is pointless